.github/workflows/go.yml

```diff
@@ -44,4 +44,6 @@ jobs:
       run: go build -v ./...
 
     - name: Test
-      run: go test -v -race ./...
+      run: | 
+        go test -v -race ./...
+        go test -tags=test ./...
```

Comment:
Added this. Specific test cases in which I re-expose the resty client to test the values